### PR TITLE
Update Description to reflect that PowerShell is for more than scripting

### LINKED ISSUE #3339: Update Description in the Extension Gallery
image

The description currently says `Develop PowerShell scripts`. I love the `Develop` part but PowerShell is so much more than scripts.

I propose `Develop PowerShell modules, commands and scripts`. I looked at Python and other languages for inspiration but Python says `Linting, Debugging (multi-threaded, remote), Intellisense, Jupyter Notebooks, code formatting, refactoring, unit tests, and more.` Perhaps something along those lines are better. There doesn't seem to be a standard.

### PR DESCRIPTION
**Pull request by potatoqualitee**
## PR Summary

Updates the description to reflect that PowerShell is a powerful language. Fixes #3339.

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [X] Summarized changes
- [ ] PR has tests
- [X] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

**Comment by SydneyhSmith** [comment]
LGTM, thanks for the PR @potatoqualitee

File: tools/ReleaseTools.psm1 [hunk 1]
```diff
@@ -271,7 +271,7 @@ function Update-Version {
     # TODO: Maybe cleanup the replacement logic.
     switch ($RepositoryName) {
         "vscode-powershell" {
-            $d = "Develop PowerShell scripts in Visual Studio Code!"
+            $d = "Develop PowerShell modules, commands and scripts in Visual Studio Code!"
```
Comment (andschwa) [inline tools/ReleaseTools.psm1:274]: Thanks for catching this 😁

File: vscode-powershell.build.ps1 [hunk 2]
```diff
@@ -141,12 +141,12 @@ task UpdatePackageJson {
     if ($script:IsPreviewExtension) {
         $script:PackageJson.name = "powershell-preview"
         $script:PackageJson.displayName = "PowerShell Preview"
-        $script:PackageJson.description = "(Preview) Develop PowerShell scripts in Visual Studio Code!"
+        $script:PackageJson.description = "(Preview) Develop PowerShell modules, commands and scripts in Visual Studio Code!"
```
Comment (andschwa) [inline vscode-powershell.build.ps1:144]: Omg what is this script doing. @rjmholt?!

**Review by andschwa — APPROVED** [review]
Thanks @potatoqualitee!

**Pull request merged**